src/client.js

```diff
@@ -1777,6 +1777,27 @@ class Client {
         const result = this._toRepresentation(doc);
         return result;
     }
+
+    /**
+     * Creates and returns a new instance of the specified entity
+     * @param entityType
+     * @param entity
+     * @returns {Promise<XML.XtkObject>}
+     */
+    async newInstance(entityType, entity) {
+        const that = this;
+        const root = entityType.split(":").pop();
+        const xml= this._fromRepresentation(root, entity, 'SimpleJson')
+        const soapCall = this._prepareSoapCall(entityType, "NewInstance", false, this._connectionParameters._options.extraHttpHeaders);
+        soapCall.writeDocument('entity', xml);
```

Comment:
What happen in case of errors? is there a layer that throw or catch the exception? is this handled and tested